### DISCUSSION TITLE
Reading Grid Problem

### DISCUSSION THREAD

**Discussion by Dewu-Yang** [loganoz/horses3d#171] [Q&A]
Dear all,
   When I encrypt the grid, the following error occurs. I was confused. The grid works fine with sparse points, but once the number of grid points increases, this error appears. I would greatly appreciate it if you could provide me some advice!

_forrtl: severe (111): position number is outside of the valid range, unit -265, file /dssg/home/acct-saaxh/saaxh/YDW/horses3d/Solver/test/NavierStokes/Blade/RESULTS/blade_0000000000.hsol
Image              PC                Routine            Line        Source             
libifcoremt.so.5   000014C9F26E0E79  for__io_return        Unknown  Unknown
libifcoremt.so.5   000014C9F274793C  for_write_seq         Unknown  Unknown
horses3d.ns        0000000000861FEC  Unknown               Unknown  Unknown
horses3d.ns        00000000007AC56D  Unknown               Unknown  Unknown
horses3d.ns        000000000058AFAA  Unknown               Unknown  Unknown
horses3d.ns        0000000000433A77  Unknown               Unknown  Unknown
horses3d.ns        0000000000404812  Unknown               Unknown  Unknown
libc-2.28.so       000014C9EC8ECCF3  __libc_start_main     Unknown  Unknown
horses3d.ns        000000000040471E  Unknown               Unknown  Unknown_
 
Best,
Dewu

**Comment by loganoz** [comment]
Hi @Dewu-Yang, 

Sorry for the delay in the response, I have been out of office these weeks. 
I think we have some issues with meshes with a very big number of elements. How many elements does your mesh have? 

Best,
Gonzalo

**Reply by Dewu-Yang** [reply]
Dear Prof.Gonzalo,

 Thank you very much for your response. The quantity of my h-type grid is around twenty thousand. The grid encounters errors when I set p≥5. Lowering the order is computationally feasible. It is expected that the computation termination occurs when high polynomial orders are used due to excessively small sizes at the finest grid locations.

Best,
Dewu

**Reply by loganoz** [reply]
I think that is something different then. 
Can you compile and run in debug mode so we get more information? 
@oscarmarino, @hmarbona, @Andres-MG, @stecolumbus any ideas?

**Reply by oscarmarino** [reply]
The problem here is for very high DOF, the hsol file cannot write anymore because the integer in fortran is overflow. Do you have active the save gradients option? If it is, by disabling it you can still save results (without gradients). We are aware of this problem and will provide a solution in the near future.

**Reply by oscarmarino** [reply]
This has been fixed now with the newest version.
